README.md

```diff
@@ -0,0 +1,65 @@
+# Uyuni RFCs
+
+Many changes, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial" - those should be put through an appropriate design process and produce a consensus among the Uyuni core team.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for such substantial changes to enter the project. This process will be adjusted over time as more features are implemented and the community settles on specific approaches to feature development.
+
+## TL;DR
+
+* Fork https://github.com/uyuni-project/uyuni-rfc
+* Copy `00000-template.md` to `active/0000-my-feature.md`
+  - 'my-feature' should be as descriptive as possible.
+  - please don't assign an RFC number yet.
+* Write the content, based on the [RFC template](0000-template.md).
+* Open a Pull Request and discuss it.
+* When the discussion is finished and the PR ready to be merged, lookup the next free number in master and rename the file.
+
+Once the PR is merged, [implementation can start](#first-principles).
+
+> The Uyuni core team will work towards reviewing the set of open RFC pull requests and provide feedback on a weekly basis. All community members are invited to discuss the open pull requests.
+
+## When to follow this process
+
+You should consider using this process if you intend to make "substantial" changes to Uyuni or its documentation. Some examples of what typically requires an RFC are:
+
+   - A new feature that creates new API surface area.
+   - Architectural changes or introduction of new conventions.
+   - Any disruptive changes in the user experience.
+   - The removal of features that were already shipped.
+
+The RFC process encourages discussions about a proposed feature during its early stages, in order to incorporate important constraints into the design. It is a great opportunity to get more eyeballs on your proposal before it becomes a part of a released version of Uyuni. Quite often, even proposals that seem "obvious" can be significantly improved once a wider group of interested people have a chance to weigh in.
+
+
+
+#### Changes that do **NOT** require an RFC
+
+  - Bug fixes.
+  - Rephrasing, reorganizing or refactoring.
+  - Improvements in test coverage or documentation.
+  - UI improvements on existing features.
+
+## First principles
+
+- A RFC is considered 'active' when it gets merged into the `uyuni-rfc` repository.
+- Once an RFC becomes active, the feature can be implemented and submitted as a pull request to the Uyuni repository.
```

Comment:
I am not sure if 'active' is the right word here, maybe we should say 'accepted' or 'agreed upon' instead? This way it would still be valid even if the implementation is incrementally drifting away from the RFC over time. It was 'accepted' at the time of merge, while it will most likely not stay 'active' forever in case we don't keep it up to date with the code. Also in some cases it might happen that people want to submit a PR with a prototype at the same time together with the RFC, which would be okay I guess.

Reply:
Changed all references to 'active' to 'accepted'.